src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,32 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import usersFromServer from '../../api/users';
+import { UserInfo } from '../UserInfo';
+
+type Togo = {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+};
```

Comment:
The type `Togo` should be renamed to `Todo` to maintain consistency with the naming conventions used in a todo application.